meta/block_pass.rb

```diff
@@ -3,4 +3,6 @@
 
   singleton_mutations
   mutation 'foo'
+  mutation 'foo(&nil)'
```

Comment:
They are not invalid. I had a check for them, but I tried it, and you can actually do `&nil`. It's a no-op so we could choose not to do it if we wanted. We could also skip `&self` but that works for objects that define `#to_proc`.

```ruby
[1].map(&nil).to_a # => [1]
```